easybuild/easyconfigs/a/ABINIT/ABINIT-7.10.5-intel-2016.02-GCC-4.9.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'ConfigureMake'
+
+name = 'ABINIT'
+version = "7.10.5"
+
+homepage = 'http://www.abinit.org/'
+description = """ABINIT is a package whose main program allows one to find the total energy,
+ charge density and  electronic structure of systems made of electrons and nuclei (molecules
+ and periodic solids) within Density Functional  Theory (DFT), using pseudopotentials and a
+ planewave or wavelet basis."""
+
+toolchain = {'name': 'intel', 'version': '2016.02-GCC-4.9'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['http://ftp.abinit.org/']
+sources = [SOURCELOWER_TAR_GZ]
+
+# ensure mpi and intel toolchain
+configopts = '--with-fc-vendor=intel --with-linalg-flavor=mkl --enable-mpi '
+configopts += '--with-fft-flavor=fftw3-mkl '
+configopts += '--with-fft-libs="$LIBFFT"'
+
+# there are variants (e.g. --with-dft-flavor), but is an easyconfig file needed for all?
+# eb --try-amend can be used to install those variants from the same easyconfig file...
```

Comment:
@migueldiascosta please drop these comments, you've answered the question yourself including a separate easyconfig for `-libxc`?